charts/fission-all/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-all
-version: 1.5.0
+version: 1.6.0
 description: Fission is a fast serverless framework for Kubernetes.
```

Comment:
We will change this when creating a new release, please rollback the changes. (fission-all and fission-core)